Sources/SwiftInspectorVisitors/TypeDescription.swift

```diff
@@ -104,6 +105,24 @@ public enum TypeDescription: Codable, Hashable {
       } else {
         return "\(parentType.asSource).\(name)<\(generics.map { $0.asSource }.joined(separator: ", "))>"
       }
+    case let .attributed(type, specifier: specifier, attributes: attributes):
+      func attributesFromList(_ attributes: [String]) -> String {
+        attributes
+          .map { "@\($0)" }
```

Comment:
Per [the Swift reference manual](https://docs.swift.org/swift-book/ReferenceManual/Attributes.html#grammar_attributes), the attribute grammar always starts with an `@`:
<img width="425" alt="image" src="https://user-images.githubusercontent.com/139364/140258130-65992291-8310-41ba-a186-0993bfc07da1.png">

Reply:
Are property wrappers "attributes" in the grammar?

Reply:
Newer versions of Swift support property wrappers on function arguments https://github.com/apple/swift-evolution/blob/main/proposals/0293-extend-property-wrappers-to-function-and-closure-parameters.md.

Reply:
I believe so, yes. But they'd be `attributes` on a `***DeclSyntax` type rather than an `AttributedTypeSyntax` since property wrappers are on the type declaration rather than the type reference.

We don't currently track attributes on type declarations. We’ll Get There™ (once we find a need for them)

Reply:
The property wrappers on functions and closures look like these are attributes, but on the property rather than a type since the attribute comes before the property name. An example in that doc is `@Traceable value: Value`. If this wrapper looked like `value: @Traceable Value` then yes this would be an attribute on the type rather than the property.